eng/Publishing.props

```diff
@@ -1,8 +1,11 @@
 <Project>
 
+  <!-- Update Artifacts with Kind=Package to have additional metadata item Category="ToolingPackage".
+       Depending on channel configuration, this means that these assets could be pushed to a different feed. -->
   <ItemGroup>
     <Artifact Include="$(ArtifactsDir)nupkgs/*.nupkg"
-              Kind="Package">
+              Kind="Package"
+              Category="ToolingPackage">
       <Visibility Condition="'$(EnableDefaultRidSpecificArtifacts)' == 'true' and '$(DotNetBuildOrchestrator)' == 'true'">Vertical</Visibility>
     </Artifact>
   </ItemGroup>
```

Comment:
I think I'd prefer keeping this as is and add the `<Artifact Update="@(Artifact->WithMetadataValue('Kind', 'Package'))" Category="ToolingPackage" />` afterwards. that way we update both the packages here and the defaults from arcade

Reply:
> that way we update both the packages here and the defaults from arcade

There are no defaults from arcade in this repository as it doesn't use the Arcade SDK. All artifacts are in non expected locations which is the reason for this item group.

Reply:
done